recipes/fast_rnnt/conda_build_config.yaml

```diff
@@ -0,0 +1,8 @@
+pytorch:
```

Comment:
Why build for so many pytorch versions?

Reply:
It uses pytorch cpp api, I am not sure whether it works if using different versions for building and running. Will try it locally.

Reply:
No, it won't but we usually build only for the latest (and sometimes second to last) version of pytorch.

Reply:
Please choose up to two pytorch versions to build against.